### PR TITLE
// BO product form quantity preferences integration

### PR DESCRIPTION
**Pull request by kelu95**

### CODE REVIEW

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 1]
```diff
@@ -115,11 +115,11 @@ var displayFieldsManager = (function() {
 
 				$('#specific-price-combination-selector').removeClass('hide').show();
 				$('#form-nav a[href="#step3"]').text(translate_javascripts['Combinations']);
-				$('#product_qty_0_shortcut_div, #quantity-no-attribute, #step3_minimal_quantity').hide();
+				$('#product_qty_0_shortcut_div, #quantitys').hide();
 			} else {
 				combinations.hide();
 				$('#specific-price-combination-selector').hide();
-				$('#product_qty_0_shortcut_div, #quantity-no-attribute, #step3_minimal_quantity').show();
+				$('#product_qty_0_shortcut_div, #quantitys').show();
```
Comment (mickaelandrieu) [inline admin-dev/themes/default/js/bundle/product/form.js]: quantities you mean ?

File: src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig [hunk 2]
```diff
@@ -234,6 +234,21 @@
                 </div>
                 <div role="tabpanel" class="tab-pane panel panel-default" id="step3">
                     <div class="col-lg-9">
+                        <div class="form-group" id="quantitys" style="{% if has_combinations or form.step3.depends_on_stock.vars.value != "0" %}display: none;{% endif %}">
+                            <h4>{{ trans('Quantitys', [], 'AdminProducts') }}</h4>
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: Quantities

File: admin-dev/themes/default/js/bundle/product/form.js (hunk 1 above)
Comment (Shudrum) [inline admin-dev/themes/default/js/bundle/product/form.js]: 😆

Comment (kelu95) [inline admin-dev/themes/default/js/bundle/product/form.js]: my bad, fixed :)

**Comment by mickaelandrieu** [comment]
thank you @kelu95

**Pull request merged**